web/src/lib/utils/timeline-util.ts

```diff
@@ -19,16 +19,16 @@ export function formatGroupTitle(date: DateTime): string {
 
   // Today
   if (today.hasSame(date, 'day')) {
-    return 'Today';
+    return date.toRelativeCalendar() || 'today';
```

Comment:
```suggestion
    return date.toRelativeCalendar() || 'Today';
```

Reply:
CSS makes it title case anyway, but I've updated the code so this `|| 'today'` is no longer needed